database/models.py

```diff
@@ -22,3 +20,7 @@ class ResultArticle(models.Model):
     content = models.TextField()
 
     searches = models.ManyToManyField(SearchParameters)
+
+    @property
+    def get_node_id(self):
+        return 'google' + str(self.id)
```

Comment:
shouldn't it be 'db' instead of 'google'?

Reply:
Yes, it should